README.adoc

```diff
@@ -1,4 +1,4 @@
-# Red Hat Single Sign-On 7.3 continuous delivery (CD) Tech Preview OpenShift container image
+# Red Hat Single Sign-On Continuous Delivery (CD) Tech Preview OpenShift container image
```

Comment:
Again, maybe change to RHCC form: `Red Hat Single Sign-On Continuous Delivery (RH-SSO CD) for OpenShift container image` ?

Reply:
I'd rather keep the amount of changes minimal and leave it as it is for the moment.

Reply:
Ok, so I'm resolving this.